04/Task04.js

```diff
@@ -1,10 +1,43 @@
 import React from 'react';
+import { useHistory, Switch, Route } from "react-router-dom";
+
+import products from "../src/products.json";
+import Shop from "../src/components/Shop";
 
 const Task04 = () => {
-    return (
-        <h1>Task04</h1>
-    );
-}
+  const history = useHistory();
+  const handleChange = (event) => {
+    history.push(`/task04${event.target.value}`);
+  };
+  const sortedProducts = (order) => {
+    if (order === "/task04/price-desc") {
+      return products.sort((a, b) => b.price - a.price);
+    } else if (order === "/task04/price-asc") {
+      return products.sort((a, b) => a.price - b.price);
+    }
```

Comment:
A jakieś rozwiązanie, które pozwoli w wygodny sposób dodawać kolejne opcje bez używania `if`?
np. odpowiednie struktura danych, na podstawie, które wygenerujesz `option` oraz wykonasz sortowanie?

PS. To zadanie dodatkowe ;P

Reply:
No dobra, podjąłem wyzwanie i zaktualizowałem to zadanie :D.